### ISSUE TITLE
Request: Add ability to disable Query Validation on Document Change

### ISSUE THREAD

**Issue by Sykout** [Borvik/vscode-postgres#109]
Due to validation works by sending an EXPLAIN query to the database, it means that on every key press, it would query the database.

But due to the use case, it means that majority of the explain would error out due to syntax error, flooding the error log.
It also get exacerbated if the file has more than one query in it, like a migration script.
So a file with 10 query to try to run 10 explain query on each stoke.

So I would like to request a setting to disable the validation; or at least disable it on key press.

Else, really appreciated your plugin, pretty much allowed me to have a single tool for all my development

**Comment by jccr** [comment]
I'd like to see this too

**Comment by chrisgoddard** [comment]
This is causing me a ton of headaches too - my db error logs are all fill of errors from EXPLAIN queries

**Comment by dj-jordan** [comment]
Yes, agreed, presently logfiles are getting clouded with unnecessary verboisty as can be seen by running the following command 

 `tail -n 50 -f /var/log/postgresql/postgresql-14-main.log  `
 
 and then keeping an eye on that console when editing a postgresql file that uses the extension.  
 
 It would be super great if someone could advise a way reduce the EXPLAIN logs that are clouding real issues in the database.
 e.g. options to 1) Enable/Disable Auto Validation;   2) Validate on Key press Yes/No  would be highly beneficial
 
 It woul
